service/http.directory/src/test/webapp/test.rtf

```diff
@@ -0,0 +1,7 @@
+{\rtf1\ansi\ansicpg1252\cocoartf1348\cocoasubrtf170
```

Comment:
Im not sure where the values came from - it was supposed to be a test file for the symlinks